pytensor/tensor/nlinalg.py

```diff
@@ -335,6 +396,7 @@ class Eigh(Eig):
     def __init__(self, UPLO="L"):
         assert UPLO in ("L", "U")
         self.UPLO = UPLO
+        self.return_components = False
```

Comment:
Commenting here because it's close. Is this another of those cases where Ops shouldn't be a subclass, but should share an abstract class? In rewrites tracking Eig, Eigh will also match